controllers/v1/programsController.js

```diff
@@ -437,14 +444,16 @@ module.exports = class Programs extends Abstract {
    * @name removeEntitiesInScope
    * @param {Object} req - requested data.
    * @param {String} req.params._id - program id.
-   * @param {Array} req.body.entities - Entities to be added.
+	 * @param {Object} req.body - data to be removed.
+	 * @param {Object} req.userDetails - User details
+	 * @param {Boolean} req.query.organizations - True if we want to update organizations details.
    * @returns {Array} Program scope roles.
    */
 
   async removeEntitiesInScope(req) {
     return new Promise(async (resolve, reject) => {
       try {
-        let programDetails = await programsHelper.removeEntitiesInScope(req.params._id, req.body.entities,req.userDetails.tenantData);
+        let programDetails = await programsHelper.removeEntitiesInScope(req.params._id, req.body,req.userDetails,req.query.organizations ? req.query.organizations : false);
```

Comment:
@MallanagoudaB please include code to check if a mandatory factor entity is remove a ALL value is set by the system.